### ISSUE TITLE
Issue with genesis meta titles

### ISSUE THREAD

**Issue by Frenchomatic** [ahmedkaludi/accelerated-mobile-pages#2074]
version 9.8.5.7 - thought this had been solved the other day and would be issued for 9.5.6 but has not found itself into the latest version either.

**Comment by MohammedKaludi** [comment]
@Frenchomatic this is regarding the ticket #2050 and it is associated with https://github.com/ahmedkaludi/accelerated-mobile-pages/milestone/72

![version 0 9 85 5 unfixed issues issue 2050 ahmedkaludi_accelerated-mobile-pages - google chrome 2018-04-27 19 06 59](https://user-images.githubusercontent.com/16618009/39365281-4ed6eb08-4a4e-11e8-9e03-5596864ac7e6.png)

And the deadline of the milestone is 30th of this month, so we are planning to push the code on 1st-may. 

There was some bug regarding archive pagination and it was coming on lot of sites, so we pushed a minor immediate update that had only the fix to the pagination issue.

So I suggest you to wait till 1st and all the tickets mentioned in this milestone will be pushed.

Closing this ticket.
